serde_arrow/src/internal/serialization_ng/utils.rs

```diff
@@ -53,11 +53,14 @@ pub trait SimpleSerializer: Sized {
     }
 
     fn serialize_unit(&mut self) -> Result<()> {
```

Comment:
As the change is driven by a missing feature for JSON values, would you mind adding a corresponding test in `serde_arrow/src/test_impls/json_values.rs`? Maybe just a single test with some examplary nullable primitives (u16, f32, bool for example).

Reply:
added in local branch